lib/openshift/kata_config.rb

```diff
@@ -6,7 +6,12 @@ class KataConfig < ClusterResource
     RESOURCE = 'kataconfigs'
 
     def install_completed_node_count(user: nil, cached: true, quiet: false)
-      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'installationStatus', 'completed', 'completedNodesCount')
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'installationStatus', 'completed', 'completedNodesList')
```

Comment:
Maybe needed to use `#count` here.

```
      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'installationStatus', 'completed', 'completedNodesList')&.count || 0
```

Reply:
Otherwise seems good.

Reply:
@akostadinov, good suggestion, while doing this PR, it actually discovered a [regression](https://issues.redhat.com/browse/KATA-730).  So this method will go back to `completeNodesCount` eventually.